terraform/scripts/set-up-pingdom.sh

```diff
@@ -2,10 +2,18 @@
 
 set -eu
 TERRAFORM_ACTION=${1}
+TERRAFORM_VERSION="0.8.7"
 VERSION=0.2.2
 BINARY=terraform-provider-pingdom-tf-0.8.5-$(uname -s)-$(uname -m)
 STATEFILE=pingdom-${AWS_ACCOUNT}.tfstate
 
+#Check terraform version
+
+if [[ $(terraform version 2>&1 | head -1| sed -e 's/Terraform v//') != "${TERRAFORM_VERSION}" ]]; then
+  echo "Required Terraform version ${TERRAFORM_VERSION}"
+  exit 1
+fi
```

Comment:
This is a good idea but doesn't help if internal command is missing. I left the check in the script but added built-in in `global.tf` . Have a look.